src/fragments/start/getting-started/reactnative/setup.mdx

```diff
@@ -18,47 +18,12 @@ expo init RNAmplify
 <Block name="React Native CLI">
 
 ```bash
-npx react-native init RNAmplify
+npx react-native init amplified_todo
 ```
 
 </Block>
 </BlockSwitcher>
 
-## Initialize a new backend
```

Comment:
Clarification: This section is taken out in favor of `Generate Model` section?

Reply:
Yes.